ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml

```diff
@@ -87,8 +87,22 @@ tests:
     - chain: create-infra-move-ingress-monitoring-registry
     - chain: workers-scale
     workflow: openshift-qe-node-density-heavy
+- as: perfscale-gcp-network-policy-5nodes-periodic
```

Comment:
can we move this test to a new file?

Reply:
new file? why? It's 4.13 test.

Reply:
its a whole new section, thought we should keep this to only renaming

Reply:
Since I was removing cron jobs from 4.12, I moved it to 4.13.

Reply:
this PR is more than renaming, including cleaning up of some jobs